detox/runners/jest/DetoxStreamlineJestReporter.js

```diff
@@ -91,6 +94,16 @@ class DetoxStreamlineJestReporter extends JestVerboseReporter {
       return reporterName === 'default';
     });
   }
+
+  async onRunComplete(_contexts, { testResults }) {
+    await super.onRunComplete();
+
+    const failedFiles = testResults
```

Comment:
@d4vidi, that might be the most controversial point here, but if we add the logic here, it is going to be accessible immediately for all Detox + Jest users. 🤷‍♂️

Reply:
Oh man. Not a fan of this. I think this deserves an independent reporter.

Reply:
Okay, will do.